indexer/services/comlink/src/controllers/api/v4/compliance-v2-controller.ts

```diff
@@ -34,6 +36,11 @@ export enum ComplianceAction {
   CONNECT = 'CONNECT',
 }
 
+const complianceProgression: Partial<Record<ComplianceStatus, ComplianceStatus>> = {
```

Comment:
nit: As it's a constant config.
```suggestion
const COMPLIANCE_PROGRESSION: Partial<Record<ComplianceStatus, ComplianceStatus>> = {
```